### PR TITLE
sof-kernel-log-check:ignore i915 false failure(update)

### PR DESCRIPTION
**Pull request by RuiqingX**
Fixes: #374

Signed-off-by: RuiqingX <ruiqingx.zhang@intel.com>

### CODE REVIEW

**Comment by aiChaoSONG** [comment]
I don't fully get the point here, why add a pair of raw `*`, could you please explain a little bit?

**Comment by marc-hb** [comment]
Use `fake_kern_error()` in `case-lib/lib.sh` to locally test `ignore_str` regular expressions  when the real error is intermittent.

File: tools/sof-kernel-log-check.sh [hunk 1]
```diff
@@ -186,8 +186,8 @@ ignore_str="$ignore_str"'|acpi device:00: Failed to change power state to D3hot'
 
 # I915, issues reported by sof-test
 # BugLink: https://github.com/thesofproject/sof-test/issues/374
```
Comment (xiulipan) [inline tools/sof-kernel-log-check.sh:188]: please also put origin error logs here. A link it not enough

**Review by xiulipan — CHANGES_REQUESTED** [review]
Please add more commit message, a simple fix could not tell anything. Also the commit message should be something start with fix.

**Comment by marc-hb** [comment]
The code change is good. The commit message says basically nothing. 

Besides an English description of the change it should also contain "Fixes: 589920e6ee83"

**Review by aiChaoSONG — APPROVED** [review]
LGTM

**Pull request merged**